specs/market-api.yaml

```diff
@@ -704,26 +714,25 @@ paths:
 
         **Note**: Mutually exclusive with `rejectAgreement`.
       responses:
-        '200':
+        200:
```

Comment:
Code `200` should mean `Approved` only. No body is needed in this case.
For `Cancelled` we should use `410` gone.

Reply:
OK